CONTRIBUTING.md

```diff
@@ -19,3 +19,51 @@ accept your pull requests.
 1. Fork the repo, develop and test your code changes.
 1. Ensure that your code adheres to the existing style in the sample to which you are contributing.
 1. Submit a pull request.
+
+
+## Release guide
+
+```sh
+# * Build the latest *
+yarn
+cd lighthouse-cli; yarn; yarn run build; cd ..
+cd lighthouse-viewer; yarn; gulp; cd ..
+cd lighthouse-extension; yarn; gulp build; cd ..
```

Comment:
extension needs to be built after bumping the version (`gulp package` just packages, doesn't rebuild)